packages/flutter/test/material/banner_test.dart

```diff
@@ -100,6 +100,28 @@ void main() {
     expect(contentBottomLeft.dx, lessThan(actionsTopRight.dx));
   });
 
+  // Regression for https://github.com/flutter/flutter/issues/39574
+  testWidgets('Single action laid out beside content but aligned to the trailing edge', (WidgetTester tester) async {
```

Comment:
Would be helpful to test the RTL case as well.

Reply:
Good idea, done